edb/api/errors.txt

```diff
@@ -21,6 +21,9 @@
 
 0x_01_00_00_00   InternalServerError
 
+0x_01_00_00_01   PostgresUnavailableError
```

Comment:
I don't think this should be classified as an `InternalServerError`.  Postgres crashes may happen because of external circumstances like hardware failure, memory pressure etc.  We should add a new category, e.g. `AvailabilityError` (it could also be made retryable, I think).